addon.xml

```diff
@@ -1,9 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.coursera" name="Coursera" version="0.302" provider-name="Alex Rothberg (cancan101)">
+<addon id="plugin.video.coursera" name="Coursera" version="0.3.1" provider-name="Alex Rothberg (cancan101)">
   <requires>
     <import addon="xbmc.python" version="2.0"/>
-    <import addon="script.module.xbmcswift2" version="1.1.1"/>
-    <import addon="script.module.beautifulsoup" version="3.0.8"/>
+    <import addon="script.module.xbmcswift2" version="2.4.0"/>
```

Comment:
[over here](http://kodi.wiki/view/Add-on:Xbmcswift2) and in xbmc itself after installing xbmcswift2 manually (in settings -> add-ons -> add-on libraries)